lib/libavtp/include/avtp.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *    * Redistributions of source code must retain the above copyright notice,
+ *      this list of conditions and the following disclaimer.
+ *    * Redistributions in binary form must reproduce the above copyright
+ *      notice, this list of conditions and the following disclaimer in the
+ *      documentation and/or other materials provided with the distribution.
+ *    * Neither the name of Intel Corporation nor the names of its contributors
+ *      may be used to endorse or promote products derived from this software
+ *      without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#pragma once
+
+#include <errno.h>
+#include <stdint.h>
+
+/* AVTP subtypes values. For further information refer to section 4.4.3.2 from
+ * IEEE 1722-2016 spec.
+ */
+#define AVTP_SUBTYPE_61883_IIDC			0x00
+#define AVTP_SUBTYPE_MMA_STREAM			0x01
+#define AVTP_SUBTYPE_AAF			0x02
+#define AVTP_SUBTYPE_CVF			0x03
+#define AVTP_SUBTYPE_CRF			0x04
+#define AVTP_SUBTYPE_TSCF			0x05
+#define AVTP_SUBTYPE_SVF			0x06
+#define AVTP_SUBTYPE_RVF			0x07
+#define AVTP_SUBTYPE_AEF_CONTINUOUS		0x6E
```

Comment:
Alignment?

Reply:
Hmm, this looks aligned to me. Have you set 8-space tab?

Reply:
I'm just going by what I see in github

Reply:
I see. On vim the alignment is corret but github shows it misaligned for some reason :(

Reply:
I haven't read your coding guide. Can you just use spaces?

Reply:
Libavtp follows the coding style from the Linux kernel which uses tab for indentation.

If we take a took at the code using github UI (see https://github.com/aguedes/OpenAvnu/blob/libavtp/lib/libavtp/include/avtp.h) we can also verify the aligment is correct. In the diff it shows misaligned because of the '+' added to the beginning of the line.